ci-operator/step-registry/ipi/install/hosted-loki-staging/ipi-install-hosted-loki-staging-commands.sh

```diff
@@ -365,7 +365,9 @@ spec:
   updateStrategy:
     type: RollingUpdate
 EOF
-cat >> "${SHARED_DIR}/manifest_psp.yml" << EOF
+ocp_minor_version=$(oc version -o json | jq -r '.openshiftVersion' | cut -d '.' -f2)
+if [ "$ocp_minor_version" -le "11" ]; then
+  cat >> "${SHARED_DIR}/manifest_psp.yml" << EOF
```

Comment:
why did we have these in the first place/what are we losing by dropping them?

were we just constraining loki's security permissions to be good citizens?

Reply:
CC @vrutkovs